src/main/scala/com/codacy/client/stash/Permission.scala

```diff
@@ -0,0 +1,13 @@
+package com.codacy.client.stash
+
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+final case class Permission(user: User, permission: String)
+
+object Permission {
+  implicit val reader: Reads[Permission] = {
+    ((__ \ "user").read[User] and
+      (__ \ "permission").read[String])(Permission.apply _)
+  }
```

Comment:
Here it should be possible to use:
```suggestion
  implicit val reader: Reads[Permission] = Json.reads[Permission]
```

Reply:
Agreed, but following existing implementation for consistency in the code base.